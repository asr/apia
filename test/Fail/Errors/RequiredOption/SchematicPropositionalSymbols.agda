------------------------------------------------------------------------------
-- Testing the --schematic-propositional-symbols option
------------------------------------------------------------------------------

{-# OPTIONS --exact-split              #-}
{-# OPTIONS --no-sized-types           #-}
{-# OPTIONS --no-universe-polymorphism #-}
{-# OPTIONS --without-K                #-}

-- Fails because requires the above option.

module RequiredOption.SchematicPropositionalSymbols where

postulate D : Set

postulate id : (P : Set) → P → P
{-# ATP prove id #-}
