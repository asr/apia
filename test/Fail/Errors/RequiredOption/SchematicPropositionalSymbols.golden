ret > ExitFailure 1
err > apia: use the ‘--schematic-propositional-symbols’ option for the translation of first-order logic universal quantified propositional symbols
err >
